src/common/transformations/src/transformations/fp16_compression/align_mixed_fp32_fp16_types.cpp

```diff
@@ -16,6 +16,17 @@ using namespace ov;
 
 bool ov::pass::AlignMixedFP32FP16Types::run_on_model(const std::shared_ptr<ov::Model>& model) {
     RUN_ON_MODEL_SCOPE(AlignMixedFP32FP16Types);
+    std::unordered_set<std::string> new_friendly_names;
+
+    auto generate_uniq_name = [&new_friendly_names](const std::string& initial_name) {
```

Comment:
I am just curious. Why we call set_friendly_name() instead of using default generated name?

Reply:
I think it was done to increase readability.